core/src/test/scala/org/apache/spark/rpc/akka/AkkaRpcEnvSuite.scala

```diff
@@ -47,4 +57,71 @@ class AkkaRpcEnvSuite extends RpcEnvSuite {
     }
   }
 
+  test("timeout on ask Future with RpcTimeout") {
+
+    class EchoActor(sleepDuration: Long) extends Actor {
```

Comment:
Is it possible to write this test using the new Rpc API? Then we don't need to rewrite this test when removing Akka.